src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -536,13 +536,10 @@
 helps['acr task run'] = """
     type: command
     short-summary: Queues a quick run providing streamed logs for an Azure Container Registry.
-    examples:
+    example:
         - name: Queue a local context, pushed to ACR with streaming logs.
           text: >
             az acr task run -r myregistry --task-file bash-echo.yaml .
```

Comment:
Not a valid example for task run, this is quickrun (Please add a help for `az acr run`). Task run queues a run against the task definition:
`az acr task run -r myregistry -n mytask`